### DISCUSSION TITLE
Detecting files generated by programs not only via shell

### DISCUSSION THREAD

**Discussion by TimMeiwald** [stefanzweifel/git-auto-commit-action#226] [Help] [answered]
Hi, I'm either just being stupid or it could be an issue hence why I've just put this as a discussion in case I'm merely stupid. 

I've been using this action to autocommit a coverage report generated with coverage in python. But I can only seem to get it to autocommit by using cat coverage.xml > Coverage.xml where the first file(coverage.xml) was created by coverage. For some reason autocommit won't pick up files created directly by a program(coverage in this case) as opposed to shell commands. 

Is this expected behaviour? If no how would one go about getting autocommit to actually detect said file and autocommit it.

**Comment by stefanzweifel** [comment, marked as answer]
This is definitely not the expected behaviour.
Could it be that `coverage.xml` is added to your `.gitignore`-file?

If not, could you share your workflow or at least the options you pass to git-auto-commit? Do you use any "special" options like `file_pattern` or `status_options`?

**Reply by TimMeiwald** [reply]
It turns out you're right it was in the .gitignore. When I changed the output name it worked. The dangers of copy-pasting code. Thanks anyway

**Reply by stefanzweifel** [reply]
Can happen to anyone. Will add a note to the README about this.
